jquery.fancytree/index.d.ts

```diff
@@ -199,7 +202,7 @@ declare namespace Fancytree {
         /** Display name (may contain HTML) */
         title: string;
         /** Contains all extra data that was passed on node creation */
-        data: Object;
+        data: any;
```

Comment:
It's better to ask "why Object is necessary here" :). Actually `data` is anything associated with a node. 
So if I put some object into it in one place:
```
tree.rootNode.data = { model: someDomainObject }
```
then I'd expect it available in another:
```
let model = tree.rootNode.data.model;
```
but it won't compile:
TS2339: Property 'model' does not exist on type 'Object'.